### ISSUE TITLE
CPU usage for the processes is wrong on Windows

### ISSUE THREAD

**Issue by vitalipf** [sebhildebrandt/systeminformation#586]
**Describe the bug**
CPU usage for individual processes is reported around 0%. It seems broken starting with the version 5.8.0. It is ok in 5.7.14.

**To Reproduce**
var si = require('systeminformation');
si.processes()
.then(processes => {
console.log(processes.list[5].cpu);
});

**Current Output**
Around 0% always.

**Expected behavior**
Real CPU usage for the process like in Task Manager.

**Environment (please complete the following information):**
 - systeminformation package version: 5.8.0 and higher.
 - OS: Windows.

**Comment by sebhildebrandt** [comment]
@vitalipf thank you, need to go deeper into that. This came from a pull request and I have to make an investigation, how to fix this again and also tackle the problem that the contributor wanted to solve. Will take a week, do not have time the next 3-4 days.

**Comment by aksakalli** [comment]
BTW the reference table still states `pcpu` instead of `cpu` and `pmem` instead of `mem`.

**Comment by sebhildebrandt** [comment]
@aksakalli @vitalipf : i made some improvements calculating the process load - can you test it on your side? I also corrected the documentation in the README. Version 5.10.5 just published.

**Comment by vitalipf** [comment]
@sebhildebrandt , just tried version 5.10.6 and I don't see any improvements. CPU load is still reported around 0 iso real vlues reported by task manager.

**Comment by aksakalli** [comment]
The doc is updated for `cpu` but `mem` is still states `pmem` in [v5.10.5](https://github.com/sebhildebrandt/systeminformation/blob/v5.10.5/README.md?plain=1#L387).

**Comment by sebhildebrandt** [comment]
@aksakalli : thank you, now documentation should be fine.

**Comment by sebhildebrandt** [comment]
@vitalipf I reverted the PR #560 so we should have correct values back. I could also now verify the results. Can you once again check it? Thank you so much for your patience.

**Comment by vitalipf** [comment]
@sebhildebrandt , now it works as expected. Thank you very much!

**Comment by vitalipf** [comment]
@sebhildebrandt , no, I was wrong. Now it shows some huge numbers sometimes. I still have to stay on 5.7.14 it seems.
Is it possible to integrate a fix for #585 into 5.7 and release 5.7.15?

**Comment by sebhildebrandt** [comment]
@vitalipf Maybe it makes more sense to have the exact same code for the processes in the current version as it was in 5.7.14. I thought I did reverting it back correctly but it seems, that I made a mistake here? It would be better to have the current version fixed instead of an older version.

**Comment by vitalipf** [comment]
It will be difficult to have exact same code as you started to use powershell iso wmi at some point. Btw, on Windows Server 2016 powershell.exe is not in PATH by default so that the functionality does not work at all. It can be fixed by adding a new entry into PATH variable. And then we come back to wrong CPU usage numbers.

**Comment by sebhildebrandt** [comment]
@vitalipf I will got through it this weekend. Sorry for the mess ...
